File: Python/1113.py
# -*- coding: utf-8 -*-
while True:
    x, y = input().split()
    x = int(x)
    y = int(y)

    if(x > y):
        print("Decrescente")
    elif x < y:
        print("Crescente")
    elif(x == y):
        break

File: Java/1113.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);

        while (true) {
            int x = leitor.nextInt();
            int y = leitor.nextInt();

            if (x > y) {
                System.out.println("Decrescente");
            } 
            else if (x < y) {
                System.out.println("Crescente");
            } 
            else if (x == y) {
                break;
            }
        }
    }
 
}